android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationError.kt

```diff
@@ -72,6 +72,14 @@ sealed class TranslationError(
     class CouldNotRestoreError(override val cause: Throwable?) :
         TranslationError(errorName = "could-not-restore", displayError = false, cause = cause)
 
+    /**
+     * Could not determine translation download size between two pairs.
```

Comment:
```suggestion
     * Could not determine the translation download size between the translation pairs.
```
Could you also include what the "pairs" are

Reply:
Rephrased, ty!